StitchCore/StitchCore/Source/Core/StitchClient.swift

```diff
@@ -168,11 +167,32 @@ public class StitchClient: StitchClientType {
             - networkAdapter: Optional interface if AlamoFire is not desired.
      */
     fileprivate init(appId: String,
-                     baseUrl: String = Consts.DefaultBaseUrl,
-                     networkAdapter: NetworkAdapter = StitchNetworkAdapter()) {
+                     baseUrl: String = "https://stitch.mongodb.com",
+                     networkAdapter: NetworkAdapter = StitchNetworkAdapter(),
+                     storage: Storage? = nil) {
         self.appId = appId
         self.baseUrl = baseUrl
         self.networkAdapter = networkAdapter
+
+        let suiteName = "\(Consts.UserDefaultsName).\(appId)"
+        if let storage = storage {
+            self.storage = storage
+        } else {
+            #if !os(Linux)
+            guard let userDefaults = UserDefaults.init(suiteName: suiteName) else {
```

Comment:
[style q] I saw this in several places, is there any reason we use the explicit `.init()` call rather than just calling the init function like `UserDefaults(suiteName: blah)`?

Reply:
I believe there's actually a difference where not specifying `init` causes it do be a coercion that calls init, not directly an init. I'm sure it's compiled to exactly the same thing though.

Reply:
I believe @edaniels is correct. Personally, I do it because I find it to be more legible, and speaks closer to Swift's idioms– verbosity and readability over shortcuts.